src/components/Article.jsx

```diff
@@ -0,0 +1,33 @@
+// create outline of a single article and inport the id number of the article you want to accsess
+// use params and hooks
+
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export default function Article() {
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
```

Comment:
Please include a loading state and display a message to the user while the article is loading.